regionmask/defined_regions/natural_earth.py

```diff
@@ -213,5 +216,37 @@ def land_110(self):
             self._land_110 = _obtain_ne(**opt)
         return self._land_110
 
+    @property
+    def ocean_basins_50(self):
+        if self._ocean_basins_50 is None:
+
+            opt = dict(
+                resolution="50m",
+                category="physical",
+                name="geography_marine_polys",
+                title="Natural Earth: ocean basins 50m",
+                names="name",
+                abbrevs="name",
+            )
+
+            self._ocean_basins_50 = _obtain_ne(**opt)
```

Comment:
For the Mediterranean I would do it like so:

```suggestion
            regs = _obtain_ne(**opt)
            
            # NOTE: naturalearth includes duplicate names

            # raise an error if naturalearth changes the name
            msg = "naturalearth renamed this region, please raise an issue in regionmask"
            assert regs[14].name == 'Mediterranean Sea' , msg

            regs[14].name = 'Mediterranean Sea East'                                                                                                                                                    
            regs[14].abbrev = 'Mediterranean Sea East'
            # dito for regs[30] 
            self._ocean_basins_50 = regs
```

Reply:
This seems like a great workaround until they fix it upstream.

Reply:
For the time beeing. Should we just do the same thing with the Ross Sea? Ill raise an issue upstream.

Reply:
Upstream issue here: https://github.com/nvkelso/natural-earth-vector/issues/319